src/sidebar/services/frame-sync.js

```diff
@@ -8,29 +8,33 @@ import { isReply, isPublic } from '../helpers/annotation-metadata';
 import { watch } from '../util/watch';
 
 /**
+ * @typedef {import('../../types/annotator').AnnotationData} AnnotationData
+ * @typedef {import('../../types/api').Annotation} Annotation
  * @typedef {import('../../types/bridge-events').SidebarToHostEvent} SidebarToHostEvent
  * @typedef {import('../../types/bridge-events').HostToSidebarEvent} HostToSidebarEvent
  * @typedef {import('../../types/bridge-events').SidebarToGuestEvent} SidebarToGuestEvent
  * @typedef {import('../../types/bridge-events').GuestToSidebarEvent} GuestToSidebarEvent
  * @typedef {import('../../shared/port-rpc').PortRPC} PortRPC
+ * @typedef {import('../store/modules/frames').Frame} Frame
  */
 
 /**
  * Return a minimal representation of an annotation that can be sent from the
- * sidebar app to a connected frame.
+ * sidebar app to the host frame.
```

Comment:
```suggestion
 * sidebar app to a guest frame.
```

Annotations are displayed in guest frames, which may or may not be a host frame.